hathor/event/storage/memory_storage.py

```diff
@@ -33,3 +33,15 @@ def get_event(self, key: int) -> Optional[BaseEvent]:
             return None
 
         return self.events[key]
+
+    def get_from_event(self, key: int, size: int) -> List[BaseEvent]:
+        events = []
+
+        for i in range(size):
+            current_event_key = key+i
```

Comment:
It's weird that the linter hasn't complained about it. Should it be `key + i` (with spaces)?

Reply:
Yeah. I'm fixing this.

Reply:
It looks like this code also doesn't exist anymore.